.babelrc

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
i have included the latest version of ream atm, not my fault it updates so often.